### ISSUE TITLE
New daily note button

### ISSUE THREAD

**Issue by RedMoikle** [Irony95/ObsidianAndroidWidget#2]
The ability to quickly add/open todays daily note (following the template set up in the obsidian settings) would be incredibly useful. Might need you to add an option to define a location for these to go in however, if you aren't directly hooking into obsidian's system.

**Comment by evelynwightman** [comment]
Yes! Related: I'd love to be able to have the widget display today's daily note without having to re-set it manually every day.
